src/models/jwtPermissions/index.js

```diff
@@ -136,3 +155,48 @@ exports.init = async function ({ publicKeyUrl = _publicKeyUrl, loadPublicKeyFrom
         }
     }, 60 * 60 * 1000);
 };
+
+/**
+ *
+ * @param {string}tokenMerchantId
+ * @param {string}uriMerchantId
+ * @returns {void}
+ */
+const validateMerchant = (tokenMerchantId, uriMerchantId) => {
+    if (tokenMerchantId !== uriMerchantId && tokenMerchantId !== '*') {
+        throw new UserNotAuthorised(uriMerchantId);
+    }
+};
+
+/**
+ *
+ * @param ctx
+ * @param next
+ * @returns {Promise<void>}
+ */
+exports.validateMerchantMidleware = async (ctx, next) => {
+    const { jwtPayload } = ctx.state.jwtPayload;
+    const permisisonScope = getScope(jwtPayload, PERMISSION_SCOPE);
+    const tokenMerchantId = permisisonScope[2].merchantId;
+    const uriMerchantId = ctx.params.merchantId;
+    validateMerchant(tokenMerchantId, uriMerchantId);
+    await next();
+};
+
+/**
+ *
+ * @param {string}token
```

Comment:
fix the formatting, spaces before parameter names in the docs

Reply:
done

Reply:
done